sys/net/application_layer/dhcpv6/client.c

```diff
@@ -230,13 +230,13 @@ void dhcpv6_client_req_ia_pd(unsigned netif, unsigned pfx_len)
     }
 }
 
-void dhcpv6_client_req_ia_na(unsigned netif)
+int dhcpv6_client_req_ia_na(unsigned netif)
 {
     assert(IS_USED(MODULE_DHCPV6_CLIENT_IA_NA));
     if (!IS_USED(MODULE_DHCPV6_CLIENT_IA_NA)) {
         LOG_WARNING("DHCPv6 client: Unable to request IA_NA as module "
                     "`dhcpv6_client_ia_na` is not used\n");
-        return;
+        return 0; /* TODO: Is this the right return value? */
```

Comment:
```suggestion
        return -ENOTSUP;
```

?

Reply:
Thank you! I added this in 50ab3b1bb880827ab7f9e41746c0a9e60c895b63